### PR TITLE
PLATSUP-1006 Add Embed Blurb

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/PLATSUP-1006

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1703200332165859) and give us a `:review-pr:` emoji.

File: docs/_embed/benefits-embedding.md [hunk 1]
```diff
@@ -10,7 +10,11 @@ redirect_from:
 # Benefits of embedding Zapier
 
 ## Introduction
-We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6000+ integrations.
+We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6, 000+ integrations.
+
+## Allow Users to Connect Faster through accelerated sign-ups
```
Comment (marinahand) [inline docs/_embed/benefits-embedding.md:15]: Allow users to connect faster through accelerated sign-ups

([style guide](https://coda.io/d/Zapier-Content-and-Messaging-Guide_dH5LiS8Mxyt/Style-and-mechanics_suRp5#_lu_D6) is sentence case for headings)

File: docs/_embed/benefits-embedding.md [hunk 2]
```diff
@@ -10,7 +10,11 @@ redirect_from:
 # Benefits of embedding Zapier
 
 ## Introduction
-We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6000+ integrations.
+We’ve put together a list of opportunities to help your integration become more visible. Try one or all of the options here and begin connecting your users with Zapier’s 6, 000+ integrations.
```
Comment (marinahand) [inline docs/_embed/benefits-embedding.md:13]: The spacing on the 6000 looks odd, should this be 6,000+?

**Comment by cameronjp** [comment]
Thanks @marinahand ! I've pushed again

**Review by marinahand — APPROVED** [review]

**Pull request merged**